references/validationFunctions/follow_up/baseScripts/requiredWhenProgression.js

```diff
@@ -7,20 +7,25 @@
 const validation = ($row, $field, $name) => 
     (function validate() {
         let result = {valid: true, message: "Ok"};
-
         /* required field, cannot be null */
         const diseaseStatus = $row.disease_status_at_followup.trim().toLowerCase();
 
         const isRequired = diseaseStatus.match(/(progression)$/);
+        const otherStatus = ["stable", "partial remission", "no evidence of disease", "complete remission"];
 
         // checks for a string just consisting of whitespace
         const checkforEmpty = (entry) => {return /^\s+$/g.test(decodeURI(entry).replace(/^"(.*)"$/, '$1'))};
 
         if (isRequired && (!$field || checkforEmpty($field))){
             result = {valid: false, message: `'${$name}' is a required field if 'disease_status_at_followup' is of type 'progression'.` }
         }
```

Comment:
```
  else if (!isRequired && ! (!$field || checkforEmpty($field))){
            result = {valid: false, message: `'${$name}' cannot be provided ..etc etc` }
        }
```

Reply:
this should suffice, I don't think the other conditions are needed

Reply:
adding this to the requiredwhenrelapse script, removing the other else-ifs, seems to be making the tests pass